### ISSUE TITLE
[Bug] Scrollbar doesn't autoscroll when moving it to the buttom

### ISSUE THREAD

**Issue by Quantum1337** [shaag7967/multiserialviewer#4]
see:

[Screencast from 23.12.2024 10:11:18.webm](https://github.com/user-attachments/assets/fb8c0748-e84f-4d50-8126-b57fb543cfa9)

**Comment by shaag7967** [comment]
yes, this is intended behaviour. I tried it out and it is kind of annoying when you inspect the debug output, scroll manually (mouse wheel) to the bottom while you are looking for something and it automatically starts scrolling by itself, even if you are not finished yet.

**Comment by shaag7967** [comment]
as a compromise, I made it configurable. Please try it out using my feature branch:
https://github.com/shaag7967/multiserialviewer/tree/feature_settingsHandling

**Comment by shaag7967** [comment]
Fixed in 25.1.0
